foreman.te

```diff
@@ -463,35 +450,28 @@ tunable_policy(`websockify_can_connect_all',`
 # Container / Docker
```

Comment:
We've dropped foreman_docker so is this still relevant?

Reply:
Removed. If you can confirm nothing in Katello connects to docker ports just to be sure.

Reply:
Wait! For technical reason (upgrade path vs core policy) I will do this separately. There was some much pain made when we were introducing this redefined port, these are darkest things in SELinux and I would rather keep this removal separate because I am pretty sure removal will be painful too.

Reply:
https://projects.theforeman.org/issues/29957